ai-backend/docker-compose.yml

```
@@ -7,6 +7,8 @@ services:
       target: dev
     ports:
       - "8000:8000"
+    environment:
+      - OPENAI_API_BASE=http://194.12.189.112:11434
```

```suggestion
      - LLM_API_BASE=http://194.12.189.112:11434
```
It's better to rename this to `LLM_API_BASE` or something like that